src/pages/variantsSearchPage/index.tsx

```diff
@@ -1,26 +1,29 @@
-import React, { FC } from 'react';
-import SearchView from './SearchView';
-import PageContent from 'components/Layout/PageContent';
-import WorkBench from './WorkBench';
+import React from 'react';
+
 import { isKfInvestigator } from 'common/profile';
-import { EgoGroups } from 'store/userTypes';
+import PageContent from 'components/Layout/PageContent';
+
+import useUser from '../../hooks/useUser';
+
+import SearchView from './SearchView';
 import VariantStats from './VariantStats';
-import style from './VariantsSearchPage.module.scss';
+import WorkBench from './WorkBench';
 
-type OwnProps = {
-  egoGroups: EgoGroups;
-};
+import style from './VariantsSearchPage.module.scss';
 
-const VariantPage: FC<OwnProps> = ({ egoGroups }) => (
-  <PageContent title={'The Kids First Variant Database'}>
-    <div className={style.variantPageGrid}>
-      <VariantStats />
-      <WorkBench isAllowed={isKfInvestigator(egoGroups)} />
-      <div className={style.variantPageGridItemTable}>
-        <SearchView />
+const VariantPage = () => {
+  const { groups } = useUser();
+  return (
+    <PageContent title={'The Kids First Variant Database'}>
```

Comment:
no need to return

Reply:
If I remove the `return` the component will return `undefined`. Maybe I am mistaken?